engine/execution/ingestion/fetcher/fetcher.go

```diff
@@ -12,7 +12,7 @@ import (
 	"github.com/onflow/flow-go/state/protocol"
 )
 
-var onlyOnflowRegex = regexp.MustCompile(`.*\.onflow\.org:3569$`)
+var onlyOnflowRegex = regexp.MustCompile(`.*\.(onflow\.org|dapper-flow\.com):3569$`)
```

Comment:
Expanding the whitelist here so that it's guaranteed to have at least one collection node available to fetch collection from.